portfolio/src/main/java/com/google/sps/servlets/CovidDataServlet.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns covid data as a JSON object, e.g. {"2017": 52, "2018": 34}] */
+@WebServlet("/covid-data")
+public class CovidDataServlet extends HttpServlet {
+  
+  ArrayList<DateEntry> covidData = new ArrayList();
+  
+  private final class DateEntry {
+    private String date;
+    private LinkedHashMap<String, Integer> cases = new LinkedHashMap();
```

Comment:
To make this a little more readable, add a comment above the hashmap on what the key and value represent.